plugins/inputs/nsdp/nsdp_test.go

```diff
@@ -0,0 +1,73 @@
+package nsdp
+
+import (
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/parsers/influx"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+	"github.com/tdrn-org/go-nsdp"
+)
+
+func TestInitDefaults(t *testing.T) {
+	plugin := defaultNSDP()
+	err := plugin.Init()
+	require.NoError(t, err)
```

Comment:
Use more brief
```suggestion
	require.NoError(t, plugin.Init())
```
same for the other unit-tests...

Reply:
Adapted via 8c321f127f426a5c1cbdc9876b56cf4409fafce2